test/cql-pytest/test_materialized_view.py

```diff
@@ -5,10 +5,13 @@
 # Tests for materialized views
 
 import time
+import re
 import pytest
 
 from util import new_test_table, unique_name, new_materialized_view, ScyllaMetrics
-from cassandra.protocol import InvalidRequest, SyntaxException
+from cassandra.protocol import ConfigurationException, InvalidRequest, SyntaxException
+
+from typing import Any
```

Comment:
Let's not start using strong typing in cql-pytest. It's not used anywhere else in this code, and we don't run any checker to enforce it.
If you really like it I'll cave in, but later I won't accept patches doing irrelevant wholesale addition of this typing stuff to existing tests.